src/PSRule.Rules.Azure/rules/LinuxOffers.Rule.ps1

```diff
@@ -0,0 +1,4618 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+$global:LinuxOffers = @(
```

Comment:
This is temporal until we can come up with a good way to keep the information up to date. Let me know if I should add the script I used to generate it somewhere in particular

Reply:
@VeraBE Works for me. Let's talk about how we can get the list of offers from ARM. Maybe from a PowerShell call.

Ultimately if we can generate it as JSON in a CI process that would be awesome. 

This is what we have done previously for provider data exports. 

https://github.com/Azure/PSRule.Rules.Azure/blob/main/scripts/providers.psm1
https://github.com/Azure/PSRule.Rules.Azure/blob/main/.github/workflows/providers.yml